src/main/java/frc/robot/common/hardware/ColorSensorMuxed.java

```diff
@@ -57,6 +59,7 @@ private boolean setI2cPort(int port) {
     for (int p : i2cPorts) {
       if (setI2cPort(p)) {
         colors[i] = sensors.getColor();
+        Shuffleboard.addEventMarker("Sensors Activated", EventImportance.kCritical);
```

Comment:
Should probably add this same event to getProximities() as well? Since both getColors() and getProximities() will read from the sensors. 

Separate thought -- does addEventMarker record a timestamp? It would be really great to have a sense of how often we were polling the sensors. Does not have to be in this PR.

Reply:
When shuffleboard is recording, it does log a timestamp.
Not sure that we should write an event for every time the sensor is active though. 
As we are sending the color to shuffleboard.

Reply:
As it has been a bit since this PR was made, many features have been added or removed since this. There is no longer a shuffleboard event anywhere in ColorSensorMuxed, and I have added a similar feature to CDSSubsystem when the sensor uses getProximities.